### DISCUSSION TITLE
Custom command with refresh

### DISCUSSION THREAD

**Discussion by khaitranhq** [NeogitOrg/neogit#1526] [Q&A] [answered]
Hi,
I employ [toggleterm.nvim](https://github.com/akinsho/toggleterm.nvim) to execute [opencommit](https://github.com/di-sukharev/opencommit/) but I haven't found out the API to refresh `neogit`.
Btw, does neogit have a built-in solution to execute a custom command like `opencommit`?
Thanks in advance for any help!

**Comment by CKolkey** [comment]
I think if you use the git-hook built into opencommit (https://github.com/di-sukharev/opencommit/?tab=readme-ov-file#git-hook-killer-feature) it should just work.

**Reply by CKolkey** [reply, marked as answer]
But to answer your question, the api is `require("neogit").dispatch_refresh()`

**Comment by khaitranhq** [comment]
Thank you. 
I can't use git hook in some of my projects because the project policies so I have to find another solution.
